packages/paste-core/components/spinner/src/index.tsx

```diff
@@ -40,3 +52,14 @@ export const Spinner = React.forwardRef<HTMLDivElement, SpinnerProps>(
     );
   }
 );
+
+if (process.env.NODE_ENV === 'development') {
```

Comment:
Still think we should ask about whether we should be adding this in but I didn't have time to bring up at 16th minute today

Reply:
Sure I think that's a good thing to discuss, I don't think it should block this PR though.

Reply:
We should be

Reply:
We should be including `if (process.env.NODE_ENV === 'development') {` ? A few of the customization PRs have been removing this line.

Reply:
I think even if we decide to remove it for whatever reason, it shouldn't necessarily be done with the customization tickets since it's not part of the AC. That can lead us to scope creep, and loss of context/history.  I'll start a thread in our team channel now, but will resolve this convo.